lib/logger.js

```diff
@@ -0,0 +1,33 @@
+import { argumentsToString } from './format';
+
+// On the server use winston
+let winstonLogger = null;
+if (typeof window === 'undefined') {
+  const winston = require('winston');
+
+  winstonLogger = new winston.Logger({
+    transports: [new winston.transports.Console()],
+  });
+
+  require('./loggly').addTransport(winstonLogger);
+}
+
+function log(level, output) {
+  if (winstonLogger) {
+    winstonLogger.log(level, output);
+  } else {
+    console.log(`${level}: ${output}`); // eslint-disable-line no-console
```

Comment:
Should we console.level as much as possible, e.g., console.error for error level?